File: PRAK305-2210817220019-DHEA_APRILINDA_UTAMI.c
#include <stdio.h>

int main()
{
    int  input, detik, menit, jam, hari;
    scanf("%d", &input);
    if (input>=86400)
    {
    hari = input / (86400);
    jam = (input % 86400/3600);
    menit = (input % 3600/60);
    detik = input % 60 ;
    printf("%d hari %.2d:%.2d:%.2d", hari, jam, menit, detik);
    }
    else
    {
    jam = (input /3600);
    menit = (input % 3600/60);
    detik = input % 60 ;
    printf("%.2d:%.2d:%.2d", jam, menit, detik);
    }
}

File: PRAK305-2210817220019-DHEA_APRILINDA_UTAMI.py
det = int(input())
if det<=86400:
    jam = det //3600
    sisa = det % 3600
    menit = sisa //60
    detik = sisa % 60
    print ('%.2d:%.2d:%.2d' % (jam, menit, detik))
else :
    hari = det //86400
    sisa = det % 86400
    jam =  sisa //3600
    ssisa = sisa % 3600
    menit = ssisa //60
    detik = ssisa % 60
    print ('%d hari %.2d:%.2d:%.2d' % (hari, jam, menit, detik))